tool/travis.sh

```diff
@@ -26,4 +26,9 @@ echo "PASSED"
 set -e
 
 # Run the tests.
-flutter test
+echo "Running tests in each package..."
+for package in $(ls -1 packages); do
```

Comment:
I think that this would fail ShellLint.  In bash it's safer to use globs:

`for package in packages/*/; do`

Reply:
Done.